front/app/containers/Admin/pagesAndMenu/containers/CustomPages/CustomPageSettingsForm/CustomPageSettingsForm.test.tsx

```diff
@@ -174,5 +180,14 @@ describe('CustomPageSettingsForm', () => {
         expect(defaultProps.onSubmit).not.toHaveBeenCalled();
       });
     });
+
+    it('keeps showing the slug input when field is erased', () => {
+      const user = userEvent.setup();
+      render(<CustomPageSettingsForm mode={mode} {...defaultProps} />);
+
+      const slugInput = screen.getByRole('textbox', { name: 'Slug' });
+      user.clear(slugInput);
+      expect(slugInput).toBeInTheDocument();
+    });
```

Comment:
A bug I discovered while working on this ticket. The input field disappeared when you erased the slug.